### PR TITLE
Update EleonorL.json

### PR DESCRIPTION
**Pull request by EleonorL**

### CODE REVIEW

File: src/data/users/EleonorL.json [hunk 1]
```diff
@@ -12,7 +12,12 @@
             "url": "https://github.com/EleonorL/RIA-Development-with-JavaScript/blob/master/blog/entry01.md",
             "title": "First Blog Entry"
         }
```
Comment (krawaller) [inline src/data/users/EleonorL.json:14]: must add comma after here!

**Comment by EleonorL** [comment]
Yes thanks just noticed!

**Comment by krawaller** [comment]
If you push changes to your fork then this PR will update too!

**Comment by krawaller** [comment]
Ah, just saw your other PR

**Pull request closed without merging**